packages/react-instantsearch/src/ui/SortBy.tsx

```diff
@@ -42,6 +42,7 @@ export function SortBy({
         className={cx('ais-SortBy-select', classNames.select)}
         onChange={(event) => onChange(event.target.value)}
         value={value}
+        aria-label="Sort results by"
```

Comment:
This harmonizes the UI component with the Vue one. This seems useful since we don't provide a `<label>` element to label the `<select>`.

More info: https://accessibilityinsights.io/info-examples/web/select-name/

Reply:
We don't have translatable aria labels on our libraries for now, so I'm fine if we keep it static for now. We could maybe have a dedicated task listing all untranslatable aria attributes across the library to add support for them at once.

Reply:
We actually do have some translatable aria labels, for example for [`<Pagination>`](https://www.algolia.com/doc/api-reference/widgets/pagination/react/#widget-param-translations). But I agree this can be done separately.